system/threads.cpp

```diff
@@ -30,15 +31,29 @@ namespace [[cheerp::genericjs]] client {
 [[cheerp::genericjs]] client::Blob* __builtin_cheerp_get_threading_blob();
 
 [[cheerp::genericjs]] client::Worker* utilityWorker = nullptr;
+[[cheerp::genericjs]] client::Map<int, client::Worker*>* workerList = nullptr;
 FutexSpinLock futexSpinLock;
-MessageQueue<ThreadSpawnInfo> threadMessagingQueue;
+MessageQueue<QueueMessage> threadMessagingQueue;
 
 extern "C" {
 
 void _start();
 
 std::atomic<uint32_t*> mainThreadWaitAddress = 0;
 
+bool isBrowserMainThread()
+{
+	if (canUseAtomicWait == UNINITIALIZED)
```

Comment:
declare `canUseAtomicWait` as a static inside here. only this function uses it directly right?

Reply:
The previous move of this variable was messy. I declared it in impl, and then defined it in wasm and browser, but only threads needs it. I've removed all instances of it outside of this function, and made it a static as you suggested.